docs/approvals/service-now/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Service Now Integration
+description: Approvals, approvals, approvals
+position: 10
+---
+```warning
+The ServiceNow Integration feature is only available as an EAP for version 2022.2.5556 or later and it requires a feature license to be able to use it.
+```
+
+## Overview
+
+The Octopus Deploy/ServiceNow integration allows users to block the execution of specifically configured deployments unless they have a corresponding approved ServiceNow **Change Request** (CR).
+
+For configured project and environment combinations, Octopus Deploy will 
```

Comment:
This sentence appears to end abruptly :smile:

Reply:
huh, looks like I fell asleep

Reply:
I can't remember what I was going to say so I just removed the line